packages/rancher-monitoring/rancher-service-metrics/charts/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,56 @@
+{{- $chartLabel := .Values.chartLabel }}
+{{- if eq $chartLabel "" }}
+{{- $chartLabel = (index (lookup "apps/v1" "Deployment" "cattle-system" "rancher").metadata.labels "chart") }}
+{{- end }}
+{{- if ne $chartLabel "" }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: rancher
+    release: rancher
```

Comment:
Can we copy the way we do this in the Monitoring chart? e.g. add a helper template function like:

https://github.com/rancher/charts/blob/73f1cd647751650190644b23d0e41221bdfed644/charts/rancher-monitoring/templates/_helpers.tpl#L105-L113

And reference it everywhere like https://github.com/rancher/charts/blob/73f1cd647751650190644b23d0e41221bdfed644/charts/rancher-monitoring/templates/prometheus/prometheus.yaml#L9

Reply:
you can also reuse this in the selector below